### PR TITLE
fix(arc-1084): add waveform for related & similar

### PR DESCRIPTION
**Pull request by ian-emsens-shd**
https://meemoo.atlassian.net/browse/ARC-913

### CODE REVIEW

**Review by bartnaessens — APPROVED** [review]

**Comment by bertyhell** [comment]
can't you use the same asset that we already have?
public/images/waveform.svg

Or does the color have to be different? Maybe there is a way to change the fill color using css?

image

File: public/images/waveform--white.svg [hunk 1]
```diff
@@ -0,0 +1,158 @@
+<?xml version="1.0" encoding="utf-8"?>
+<!DOCTYPE svg PUBLIC "-//W3C//DTD SVG 1.1//EN" "http://www.w3.org/Graphics/SVG/1.1/DTD/svg11.dtd">
+<svg version="1.1" id="Layer_1" xmlns="http://www.w3.org/2000/svg" x="0px" y="0px"
```
Comment (bertyhell) [inline public/images/waveform--white.svg:3]: a green version of this waveform already exists, shouldn't we show the same one?

**Review by bertyhell — COMMENTED** [review]
see global remarks

File: public/images/waveform--white.svg (hunk 1 above)
Comment (ian-emsens-shd) [reply public/images/waveform--white.svg:3]: image

Reworking the existing waveform to use currentColor produces too many potential side-effects

**Review by bertyhell — APPROVED** [review]

**Pull request merged**